### PR TITLE
Feature/bc 488 show invitee availability

### PR DESCRIPTION
**Pull request by yeezick**
Accomplishes [BC-488: Add availability to meeting modal](https://bootcamper.atlassian.net/browse/BC-488?atlOrigin=eyJpIjoiZDRkODIzMTE3NzAwNDMwYzliMDQyYWYzYmMwMjVlMTEiLCJwIjoiaiJ9)
* Adds availability each member checkbox 
* Separates SelectAttendees components and stylesheets into their own files 
* Creates a directory for stylesheets

### CODE REVIEW

File: src/components/Calendar/MeetingModal/SelectAttendees.tsx [hunk 1]
```diff
@@ -37,14 +34,13 @@ export const SelectAttendees = ({
...
           <FormGroup>
-            {filteredMembers.map(currMember => (
-              <div key={`select-member-${currMember._id}`}>
-                <MemberCheckbox
-                  attendees={attendees}
-                  currMember={currMember}
-                  setAttendees={setAttendees}
-                />
-              </div>
+            {projectMembers.map(currMember => (
```
Comment (HectorIlarraza) [inline src/components/Calendar/MeetingModal/SelectAttendees.tsx:37]: Should a key be added for better optimization?

File: src/interfaces/UserInterface.ts [hunk 2]
```diff
@@ -132,49 +132,36 @@ export const timeOptions = [
 
 export type TimeOption = (typeof timeOptions)[0]
 
+//TODO: Removed string from sun interface....
```
Comment (HectorIlarraza) [inline src/interfaces/UserInterface.ts:135]: Seems like this can be removed now with type string gone no?

**Review by HectorIlarraza — CHANGES_REQUESTED** [review]

File: src/components/Calendar/MeetingModal/SelectAttendees.tsx (hunk 1 above)
Comment (yeezick) [reply src/components/Calendar/MeetingModal/SelectAttendees.tsx:37]: a key is provided in the member checkbox component itself

File: src/interfaces/UserInterface.ts (hunk 2 above)
Comment (yeezick) [reply src/interfaces/UserInterface.ts:135]: yes actually

**Review by HectorIlarraza — APPROVED** [review]

**Pull request merged**